packages/react-composites/tests/browser/call/hermetic/Reactions.test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
+/* @conditional-compile-remove(reaction) */
+import { expect } from '@playwright/test';
+/* @conditional-compile-remove(reaction) */
+import { dataUiId, isTestProfileMobile, stableScreenshot, waitForSelector } from '../../common/utils';
+/* @conditional-compile-remove(reaction) */
+import { IDS } from '../../common/constants';
+/* @conditional-compile-remove(reaction) */
+import { MockCallAdapterState } from 'common';
+
+/* @conditional-compile-remove(reaction) */
+test.describe('Reactions button tests', async () => {
+  test('Reactions button sub menu appears when button is clicked', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
```

Comment:
Instead of skipping the mobile test, below you could have something like
```ts
if (isTestProfileMobile(testInfo)) {
    await page.click(dataUiId(IDS.moreButton));
    await moreDrawerShown();
} else {
    await page.click(dataUiId(IDS.reactionButton));
    await reactionFlyoutShown()
}
```